XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,602 @@
+5. Alice can now interact with/send funds to Verity.
+
+Importantly, the KYC documents that Alice sends to Isabel can include personally identifiable or private information that's needed to verify Alice's identity, but this information is never published or stored on-chain and Verity never needs to see it. Also, other businesses that trust Isabel can accept the same credentials so Alice does not need to repeatedly re-verify for every party she wants to interact with.
+
+## 2. On-Ledger Object: `Credential`
+
+This ledger object is an on-chain representation of a credential.
+
+This object costs one owner reserve for either the issuer or the subject of the credential, depending on whether the subject has accepted it.
+
+### 2.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`LedgerEntryType`| ✔️|`string`|`UInt16`|The ledger object's type (`Credential`).|
+|`Flags`| ✔️|`number`|`UInt32`|Flag values associated with this object.|
+|`Subject`| ✔️|`string`|`AccountID`|The account that the credential is for.|
+|`Issuer`| ✔️|`string`|`AccountID`|The issuer of the credential.|
+|`CredentialType`| ✔️|`string`|`Blob`|A (hex-encoded) value to identify the type of credential from the issuer.|
+|`Expiration`| |`number`|`UInt32`|Optional credential expiration.|
+|`URI`| |`string`|`Blob`|Optional additional data about the credential (such as a link to the VC document). This field isn't checked for validity and is limited to a maximum length of 256 bytes.|
+|`OwnerNode`|✔️|`string`|`UInt64`|A hint indicating which page of the owner's owner directory links to this object, in case the directory consists of multiple pages. Note: The object does not contain a direct link to the owner directory containing it, since that value can be derived from the `Account.PreviousTxnID`.|
+|`PreviousTxnID`|✔️|`string`|`Hash256`|The identifying hash of the transaction that most recently modified this object.|
+|`PreviousTxnLgrSeqNumber`|✔️|`number`|`UInt32`|The index of the ledger that contains the transaction that most recently modified this object.|
+
+#### 2.1.1. Object ID
+
+The ID of this object will be a hash that incorporates the `Subject`, `Issuer`, and `CredentialType` fields, combined with a unique space key for `Credential` objects, which will be defined during implementation.
+
+#### 2.1.2. `Flags`
+| Flag Name | Flag Value |
+|-----------|------------|
+|`lsfAccepted`|`0x00010000`|
+
+The `lsfAccepted` flag represents whether the subject of the credential has accepted the credential. If this flag is disabled, the issuer is responsible for this ledger entry's reserve; if the flag is enabled, the subject of the credential is responsible for the reserve instead. This flag is disabled by default, but is enabled as the result of a successful `CredentialAccept` transaction, or if the secondary signing flow is used to create the credential.
```

Comment:
> , or if the secondary signing flow is used to create the credential

This sentence (at the end of the paragraph) needs to be omitted. There is only one way to create crendential objects.

Reply:
Feel free to open a PR to fix that.

Reply:
https://github.com/XRPLF/XRPL-Standards/pull/251